src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static aliveOnlyFilter() {
```

Comment:
function names should start with verb (action)
```suggestion
  static filterAliveOnly() {
```